### ISSUE TITLE
'istioctl manifest generate' has `%s` in every line of output

### ISSUE THREAD

**Issue by esnible** [istio/istio#26411]
Nearly every output line has unexpected `%s`.

To reproduce:

```
$ istioctl-1.7.0-beta.2 manifest generate -o rutabaga
Component dependencies tree: 
%sBase
  Pilot

Rendering manifests to output dir %srutabaga
Rendering: %sBase
Writing manifest to %srutabaga/Base/Base.yaml
Rendering: %sPilot
Writing manifest to %srutabaga/Base/Pilot/Pilot.yaml
Rendering: %sEgressGateways
Writing manifest to %srutabaga/Base/Pilot/EgressGateways/EgressGateways.yaml
Rendering: %sAddonComponents
Writing manifest to %srutabaga/Base/Pilot/AddonComponents/AddonComponents.yaml
Rendering: %sPolicy
Writing manifest to %srutabaga/Base/Pilot/Policy/Policy.yaml
Rendering: %sTelemetry
Writing manifest to %srutabaga/Base/Pilot/Telemetry/Telemetry.yaml
Rendering: %sCni
Writing manifest to %srutabaga/Base/Pilot/Cni/Cni.yaml
Rendering: %sIngressGateways
Writing manifest to %srutabaga/Base/Pilot/IngressGateways/IngressGateways.yaml
```

This also affects every Istio 1.6.x, but not 1.5.x.

**Comment by shamsher31** [comment]
@esnible @howardjohn I can still reproduce it with `1.7.3`.

```
[istio-1.7.3]$ ./bin/istioctl manifest generate -o $HOME
Component dependencies tree: 
%sBase
  Pilot

Rendering manifests to output dir %s/home/shaansar
Rendering: %sBase
Writing manifest to %s/home/shaansar/Base/Base.yaml
Rendering: %sPilot
Writing manifest to %s/home/shaansar/Base/Pilot/Pilot.yaml
Rendering: %sPolicy
Writing manifest to %s/home/shaansar/Base/Pilot/Policy/Policy.yaml
Rendering: %sTelemetry
Writing manifest to %s/home/shaansar/Base/Pilot/Telemetry/Telemetry.yaml
Rendering: %sCni
Writing manifest to %s/home/shaansar/Base/Pilot/Cni/Cni.yaml
Rendering: %sIngressGateways
Writing manifest to %s/home/shaansar/Base/Pilot/IngressGateways/IngressGateways.yaml
Rendering: %sEgressGateways
Writing manifest to %s/home/shaansar/Base/Pilot/EgressGateways/EgressGateways.yaml
Rendering: %sAddonComponents
Writing manifest to %s/home/shaansar/Base/Pilot/AddonComponents/AddonComponents.yaml
```

The fix is in [master](https://github.com/istio/istio/pull/26416) but not available in [release-1.7](https://github.com/istio/istio/blob/release-1.7/operator/cmd/mesh/manifest-generate.go#L149)

**Comment by shamsher31** [comment]
Cherry-picked for `1.7` https://github.com/istio/istio/pull/27933
